fastlane/Fastfile

```diff
@@ -17,15 +17,16 @@ require 'yaml'
 require 'os'
 require 'fileutils'
 
-files_with_version_number = [
-  './.version',
-  './pubspec.yaml',
-  './ios/purchases_flutter.podspec',
-  './macos/purchases_flutter.podspec',
-  './ios/Classes/PurchasesFlutterPlugin.m',
-  './android/build.gradle',
-  './android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java'
-]
+files_with_version_number = {
+    './.version' => ['{x}'],
+    './pubspec.yaml' => ['version: {x}'],
+    './ios/purchases_flutter.podspec' => ["s.version          = '{x}'"],
+    './macos/purchases_flutter.podspec' => ["s.version          = '{x}'"],
+    './ios/Classes/PurchasesFlutterPlugin.m' => ['return @"{x}"'],
+    './android/build.gradle' => ["version '{x}'"],
+    './android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java' => ['PLUGIN_VERSION = "{x}"'],
```

Comment:
Nitpick: Looks like we have a bit of a mix of using single and double quotes for the patterns... I guess we sometimes use double-quotes within the text, so it might make sense to use single-quotes?